traits/observers/_dict_change_event.py

```diff
@@ -0,0 +1,75 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" Event object for representing mutations to a dict.
+"""
+
+# DictChangeEvent is exposed in the public API
+
+
+class DictChangeEvent:
+    """ Event object to represent mutations on a dict.
```

Comment:
To avoid accidents, I think it's worth adding a explicit sentence here about the difference between this API and the `removed / added / changed` API. Otherwise I think people will glance at this, and just assume that it behaves like the existing API.

Reply:
SGTM